AssemblyVersion.cs

```diff
@@ -9,4 +9,4 @@
 [assembly: CLSCompliant(true)]
 
 // Edit these for each release + update dependencies in .nuspec files
-[assembly: AssemblyInformationalVersion("0.14.22-beta")]
+[assembly: AssemblyInformationalVersion("0.14.24-beta")]
```

Comment:
Please exclude this change. We typically change this value in many more places than this (e.g. .nuspec files), and usually in the master branch, since there may be other changes included by the time this PR gets packaged.